tests/ws_integration.rs

```diff
@@ -1,1367 +1,1403 @@
-// cargo test --package surreal --bin surreal --no-default-features --features storage-mem --test ws_integration -- --nocapture
-
+// RUST_LOG=warn cargo make ci-ws-integration
 mod common;
```

Comment:
Skip this file during review: only moved the tests into `mod ws_integration` so it's easier to scope down the `cargo test` command